app/controllers/website_members_controller.rb

```diff
@@ -1,14 +1,12 @@
 class WebsiteMembersController < ApplicationController
 	load_and_authorize_resource
   def index
-    @members = current_website.website_members
+    @website_members = current_website.website_members.where("user_id IS NOT NULL")
   end
 
   def destroy
     @website_member.destroy
     return false unless @website_member.errors.full_messages.blank?
-    respond_to do |format|
-      format.js {render inline: 'location.reload();' }
-    end
+    reload_page
```

Comment:
use redirect_to here instead of the reload_page method. We should always know where to redirect after getting into an action.